app/models/user.rb

```diff
@@ -1,5 +1,7 @@
 class User < ApplicationRecord
+  has_and_belongs_to_many :tasks, join_table: :assignments
```

Comment:
Why are we choosing has_and_belongs_to_many and not has_many through?
Can you explain me the difference ?